Orm/Xtensive.Orm/Orm/Rse/Providers/Compilable/OrderProviderBase.cs

```diff
@@ -66,11 +66,11 @@ protected override void Initialize()
       var comparisonRules = new ComparisonRules[Order.Count];
       for (int i = 0; i < Order.Count; i++) {
         var orderItem = Order[i];
-        var culture = CultureInfo.InvariantCulture;
         var column = Header.Columns[orderItem.Key];
-        var mappedColumn = column as MappedColumn;
-        if (mappedColumn != null && mappedColumn.ColumnInfoRef != null)
-          culture = mappedColumn.ColumnInfoRef.CultureInfo;
+        var columnInfoRef = (column as MappedColumn)?.ColumnInfoRef;
```

Comment:
Can we make `ColumnInfoRef` a `ref readonly` property? So the code would be
```csharp
// Declaration
public class MappedColumn : Column
{
  private readonly ColumnInfoRef _columnInfo;
  public ref readonly ColumnInfoRef ColumnInfoRef => ref _columnInfo;
}

// Usage
var column = Header.Colulmns[orderItem.Key];
var culture = (column is MappedColumn mappedColumn ? mappedColumn.ColumnInfoRef.Culture : null) ?? CultureInfo.InvariantCulture;
```

Reply:
done